src/pages/en/guides/rss.md

```diff
@@ -6,11 +6,143 @@ description: An intro to RSS in Astro
+    items: import.meta.glob('./blog/**/*.md'),
+  });
+```
+
+See [Vite's glob import documentation](https://vitejs.dev/guide/features.html#glob-import) for more on this import syntax.
+
+#### 2. List of RSS feed objects
+
+We recommend this option for `.md` files outside of the `pages` directory. This is common when generating routes [via `getStaticPaths`](/en/reference/api-reference/#getstaticpaths). 
+
+For instance, say your `.md` posts are stored under a `src/posts/` directory. Each post has a `title`, `pubDate`, and `slug` in its frontmatter, where `slug` corresponds to the output URL on your site. We can generate an RSS feed using [Vite's `import.meta.globEager` helper](https://vitejs.dev/guide/features.html#glob-import) like so:
+
+```js
+// src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+
+const postImportResult = import.meta.globEager('../posts/**/*.md');
+const posts = Object.values(postImportResult);
+
+export const get = () => rss({
+    title: 'Buzz’s Blog',
+    description: 'A humble Astronaut’s guide to the stars',
+    items: posts.map((post) => ({
+      link: post.frontmatter.slug,
+      title: post.frontmatter.title,
+      pubDate: post.frontmatter.pubDate,
+    }))
+  });
+```
+
+### Adding a stylesheet
+
+You can style your RSS feed with XSL for a more pleasant user experience when viewing in your browser.
+
+Use the `rss` function's `stylesheet` option to specify an absolute path to your stylesheet. If you don't have an RSS stylesheet in mind, you can generate one by adding an `rss-styles.xsl.js` route (or whichever route you prefer) under your project's `src/pages/` directory. Here, you can apply our `getStylesheet` helper. This will serve the [Pretty Feed](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl) stylesheet when visiting `/rss-styles.xsl`:
+
+```js
+// src/pages/rss-styles.xsl.js
+import { getStylesheet } from '@astrojs/rss';
+
+export const get = getStylesheet;
+```
+
+Then, set your RSS feed's `stylesheet` option based on this file path:
+
+```js
+// src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+
+export const get = () => rss({
+    title: 'Buzz’s Blog',
+    description: 'A humble Astronaut’s guide to the stars',
+    stylesheet: '/rss-styles.xsl',
+    ...
+  });
+```
+
+If you have an XSL-based stylesheet already, place it under your project's [`public/` directory](/en/core-concepts/project-structure/#public) and set the `stylesheet` option like so:
+
+```js
+// src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+
+export const get = () => rss({
+    title: 'Buzz’s Blog',
+    description: 'A humble Astronaut’s guide to the stars',
+    // ex. use your stylesheet from "public/rss/styles.xsl"
+    stylesheet: '/rss/styles.xsl',
+    ...
+  });
+```
+
+## Using `getStaticPaths()`
+
+> **Note:** this approach is more limited compared to `@astrojs/rss`. We mainly include these instructions for legacy purposes.
```

Comment:
I'd rephrase this a bit: 

```suggestion
> **Note:** This method has been deprecated, and will be removed before the official `v1.0.0` release. Please use `@astrojs/rss` instead.
```

That was my understanding of what we were doing here, at least. Happy to chat through more if needed!

Reply:
I... didn't know if we'd do that! Sounds good if so 👍